src/components/Tabs/Tabs.tsx

```diff
@@ -40,10 +40,10 @@ export type TabsProps = {
     onChange?: (tabId: string) => void;
 };
 
-const paddingMap: Record<TabsPaddingX, string> = {
-    [TabsPaddingX.Small]: 'tw-pl-s',
-    [TabsPaddingX.Medium]: 'tw-pl-m',
-    [TabsPaddingX.Large]: 'tw-pl-l',
+const TABS_PADDING_MAP: Record<TabsPaddingX, string> = {
```

Comment:
```
const paddingMap: Record<TabsPaddingX, string> = {
    [TabsPaddingX.XSmall]: 'tw-pl-xs',
    [TabsPaddingX.Small]: 'tw-pl-s',
    [TabsPaddingX.Medium]: 'tw-pl-m',
    [TabsPaddingX.Large]: 'tw-pl-l',
};
```

Reply:
@dominiquemb In the Design, XSmall should equal 8px (as the option was named "8") so it's fine the way it is. I think the design in Figma is kind of misleading. So I would just add a fifth option:
```
const paddingMap: Record<TabsPaddingX, string> = {
    [TabsPaddingX.None]: 'tw-pl-0',
    [TabsPaddingX.XSmall]: 'tw-pl-xs',
    [TabsPaddingX.Small]: 'tw-pl-s',
    [TabsPaddingX.Medium]: 'tw-pl-m',
    [TabsPaddingX.Large]: 'tw-pl-l',
};
```

And add it to the Type.